core/src/main/scala/org/apache/spark/ui/ConsoleProgressBar.scala

```diff
@@ -40,7 +40,7 @@ private[spark] class ConsoleProgressBar(sc: SparkContext) extends Logging {
 
   // The width of terminal
   private val TerminalWidth = if (!sys.env.getOrElse("COLUMNS", "").isEmpty) {
-    sys.env.get("COLUMNS").get.toInt
+    sys.env("COLUMNS").toInt
```

Comment:
Instead of just following what your IDE says, you could actually have simplified this whole block of code.